src/lib/youtube.ts

```diff
@@ -11,32 +11,8 @@ export interface BasicInfo {
 	width: number
 }
 
-// From:
-// https://github.com/google/closure-library/blob/master/closure/goog/date/date.js
-const isoRegex = new RegExp(
-	'^(-)?P(?:(\\d+)Y)?(?:(\\d+)M)?(?:(\\d+)D)?' +
-		'(T(?:(\\d+)H)?(?:(\\d+)M)?(?:(\\d+(?:\\.\\d+)?)S)?)?$'
-)
-
-function convertDuration(isoDuration: string): number | undefined {
-	const parts = isoDuration.match(isoRegex)
-	if (!parts) {
-		return
-	}
-	// Abort if the duration specifies either year or month components.
-	if (parts[2] || parts[3]) {
-		return
-	}
-	let duration = 0
-	duration += parseFloat(parts[8]) || 0
-	duration += (parseInt(parts[7], 10) || 0) * 60
-	duration += (parseInt(parts[6], 10) || 0) * 3600
-	duration += (parseInt(parts[4], 10) || 0) * 86400
-	// Accept the leading minus sign for now, but might want to abort in future.
-	if (parts[1]) {
-		return -duration
-	}
-	return duration
+function convertDuration(isoDuration: string): string {
+	return isoDuration.substr(2).toLowerCase()
```

Comment:
Is this format good enough?

Reply:
Yeah, it's consistent with how they label the other course durations.